### DISCUSSION TITLE
Force push comparison

### DISCUSSION THREAD

**Discussion by ghost** [community/community#40963] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Hi guys,

Just wanted to say thank you for this excellent feature. I love my commit history nice, clean and well organized so I often go for interactive rebasing and subsequent force pushes when incorporating comments in my pull requests. Your new feature makes an indispensable tool for the reviewers of such unorthodox practices.

![image](https://user-images.githubusercontent.com/78210851/206019352-5fa153d4-843a-427f-8720-d0763d9dbeba.png)

It's really awesome. Thank you.
P.S.: there is an existing discussion #3478 related to this feature where further improvements are suggested, but I just wanted to give you a quick pat on the back without cluttering it up. Nice work, keep it up 👍.

**Comment by mohamedzhioua** [comment]
that's true
